city_scrapers/spiders/chi_southwest_home_equity_i.py

```diff
@@ -0,0 +1,173 @@
+import re
+from datetime import datetime, timedelta
+from io import BytesIO
+
+import requests
+from city_scrapers_core.constants import COMMISSION
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+from pdfminer.high_level import extract_text
+
+MONTH_REGEX = (
+    r"(?:Jan(?:uary)?|Feb(?:ruary)?|Mar(?:ch)?|Apr(?:il)"
+    r"?|May|June?|July?|Aug(?:ust)?|Sep(?:tember)"
+    r"?|Oct(?:ober)?|Nov(?:ember)?|Dec(?:ember)?)"
+)
+
+TIME_REGEX = r"(?:[1-9]|1[0-2]):[0-9]{2}\s(?:AM|PM)"
+
+
+class ChiSouthwestHomeEquityISpider(CityScrapersSpider):
+    name = "chi_southwest_home_equity_i"
+    agency = "Chicago Southwest Home Equity Commission I"
+    timezone = "America/Chicago"
+    allowed_domains = ["swhomeequity.com"]
+    start_urls = ["https://swhomeequity.com/agenda-%26-minutes"]
+    location = {
+        "name": "Southwest Home Equity Assurance office",
+        "address": "5334 W. 65th Street in Chicago, Illinois",
```

Comment:
If this is static we can add the ZIP code and standardize the formatting a bit so that it's "5334 W 65th St Chicago, IL 60638"